content/en/docs/tasks/administer-cluster/coredns.md

```diff
@@ -69,6 +72,14 @@ kubeadm init --feature-gates=CoreDNS=false
 
 For versions 1.13 and later, follow the guide outlined [here](/docs/reference/setup-tools/kubeadm/kubeadm-init-phase#cmd-phase-addon).
 
+## Upgrading CoreDNS 
+
+CoreDNS is available in Kubernetes since v1.9. 
```

Comment:
Could this use a `feature-state` shortcode?

Reply:
are you suggesting something like this @sftim ?
```
{{< feature-state for_k8s_version="v1.9" state="stable" >}}
```

Reply:
That's an option, yeah. It's not always appropriate; do you think it's the right fit here @rajansandeep?

Reply:
@sftim I'm sorry, I don't understand the significance of the change suggested. What does it mean?

Reply:
Ah, sorry @rajansandeep. Here's the style guide topic on [using `feature-state` shortcodes](https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#feature-state).

It's a snag; we could merge this as-is if we want and then tidy up in a later PR.

Reply:
@sftim Thank you for the link. I guess that it wouldn't make sense to add it here as CoreDNS has progressed from alpha in 1.9 to default in 1.13. Anyways, we can clean up later if it is required.

Reply:
I'd still call out stable features, especially if they're newly stable in the last release or the one before it.
(There's no clear process for dropping those markers later though).

Reply:
And, to be clear, I'm fine with this PR getting merged.